test/fixture/trigger-worker-exception/test-fallback.js

```diff
@@ -0,0 +1,5 @@
+import test from '../../../';
+
+test(() => {
+	return Promise.reject(new Error('Hi :)'));
```

Comment:
Would look better to use async function here and throw instead. Same with the below fixture.